src/js/components/TaskMesosUrlComponent.jsx

```diff
@@ -0,0 +1,65 @@
+var React = require("react/addons");
+
+var InfoActions = require("../actions/InfoActions");
+var InfoEvents = require("../events/InfoEvents");
+var InfoStore = require("../stores/InfoStore");
+
+var TaskMesosUrlComponent = React.createClass({
+  displayName: "TaskMesosUrlComponent",
+  propTypes: {
+    className: React.PropTypes.string,
+    task: React.PropTypes.object.isRequired,
+    text: React.PropTypes.string
+  },
+
+ getInitialState: function () {
+    return {
+      info: InfoStore.info
+    };
+  },
+
+  componentDidMount: function () {
+    InfoActions.requestInfo();
+  },
+
+  componentWillMount: function () {
```

Comment:
The listener needs to be removed, if the component will unmount, to avoid confusion with abandoned handlers:
```
  componentWillUnmount: function () {
    InfoStore.removeListener(InfoEvents.CHANGE,
      this.onInfoChange);
  },
```